supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalRuntimeAttributes.scala

```diff
@@ -0,0 +1,27 @@
+package cromwell.backend.impl.local
+
+import cromwell.attributes.{ContinueOnReturnCode, RuntimeAttributesValidation}
+import wdl4s.values.WdlValue
+import scalaz._
+import Scalaz._
+import lenthall.exception.MessageAggregation
+
+object LocalRuntimeAttributes extends RuntimeAttributesValidation {
+  def apply(attrs: Map[String, WdlValue]): LocalRuntimeAttributes = {
+    val continueOnReturnCode = validateContinueOnReturnCode(attrs.get("continueOnReturnCode"))
```

Comment:
This will duplicates what is in InitializationActor... This is the shared data between actors I was talking about.